data/remnant missions.txt

```diff
@@ -791,98 +791,88 @@ mission "Remnant: Return the Samples 2"
+			`	You thank the man for explaining the scanners to you and accept the next set of lessons.`
+			`	You spend a few hours viewing some of the lessons, one of which involves watching an ancient show about a merchant captain making his way through the galaxy with an unlikely crew, re-enacted by a cast of Remnant who sing and sign their way through the dialogs with exagerated simplicity. It is oddly familiar, but you can't quite place it. You notice that it seems to have a lot of subtext about the dangers of manipulation and government secrecy, especially surrounding genetics.`
+			`	Skimming through the rest of the lessons, you notice that there is a lot more here than form the last two times. It may take some time to get through it all, but you should be fairly fluent if you practice over a couple of months.`
 
 event "remnant: sign studies complete"
 
 
 
-mission "Remnant: Learn Sign 3"
+mission "Remnant: Learn Sign Follow Up"
+	invisible
 	source 
 		government "Remnant"
-	invisible
 	to offer
 		has "event: remnant: sign studies complete"
 	on offer
 		log "Have achieved a degree of fluency in the Remnant sign language sufficient for general conversation."
-		dialog
-			`As you make your way into the Caelian spaceport, it occurs to you that one of the information director who gave you the instructional videos might appreciate how much you have learned over the past few weeks of practice. You stretch your fingers and stop by the desk, where you are able to have a simple conversation with the information director.`
-			`	"Congratulations, Captain <last>!" she signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways that just by blood. I look forward to seeing what results from your work with us." She is interrupted by the arrival of other Remnant asking her something, and she bids you farewell with an apologetic gesture.`
-			decline
+		conversation
+			`As you make your way into the spaceport, it occurs to you that one of the information director who gave you the instructional videos might appreciate how much you have learned over the past months of practice. You stretch your fingers and stop by the desk where you are able to have a simple conversation with the information director.`
+			`	"Congratulations, Captain <last>!" the director signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways that just by blood. I look forward to seeing what results from your work with us." The director is interrupted by the arrival of other Remnant asking for something and bids you farewell with an apologetic gesture.`
```

Comment:
How do you sign a spoken name?

Reply:
@jafdy: Look up 'fingerspelling' in Google.